go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/GoogleContainerTools/kpt v0.30.0
+	github.com/ernesto-jimenez/gogen v0.0.0-20180125220232-d7d4131e6607 // indirect
```

Comment:
Nit I think this will be removed when running `go mod tidy`. I think it appears when we run `make docs`, (I should do something so it doesn't write to go.mod maybe?)

Reply:
`make tidy` in fact removes it. Is there a way to force a check?

Reply:
hah: https://github.com/golang/go/issues/27005#issuecomment-564892876

Reply:
One solution would be to run `tidy` and `fmt`, then fail if `git status --porcelain` gives output in the github workflow